packages/snjs/lib/device_interface.ts

```diff
@@ -34,7 +34,7 @@ export abstract class DeviceInterface {
     this.interval = null;
   }
 
-  abstract async getRawStorageValue(key: string): Promise<string | undefined>;
+  abstract getRawStorageValue(key: string): Promise<string | undefined>;
```

Comment:
Why is async removed? React Native uses AsyncStorage, which is fully async.

Reply:
Oh nevermind, back to the return type vs annotation argument 😓

Reply:
In this case it's because TypeScript marks it as an error 😅 I don't know

Reply:
https://stackoverflow.com/questions/65355164/error-ts1243-async-modifier-cannot-be-used-with-abstract-modifier